tests/e2e/features/artifacts/assetsPage/artifacts-emptyWallet.feature

```diff
@@ -36,6 +36,30 @@ Feature: Artifacts - UI
       | mainnet | withdraw    |
       | goerli  | withdraw    |
 
+    @id1618
+    Scenario: Check the Empty wallet Artifacts on the Bridge Deposit Page
+      Given I go to page "/bridge?network=era-goerli"
+      When I click by text "Deposit"
+      When I choose "ETH" as token and insert "0" as amount
+      When Element with "text" " Insufficient " should be "visible"
+      Then Message "ETH" should be visible
+      Then Message " balance on Ethereum Goerli Testnet to cover the fee. We recommend having at least " should be visible
+      Then Message " on Goerli Testnet for deposit. " should be visible
+      Then Element with "text" "Fee:" should be "invisible"
+      #Fee component
+      Then Element with "data-testid" "fee-amount" should be "invisible"
+
```

Comment:
An extra step 3 from the documentation is missed or an extra, please update doc or code.

Reply:
Fixed